en/docs/install-and-setup/setup/api-controller/managing-integrations-with-ctl.md

```diff
@@ -0,0 +1,1405 @@
+# Managing Integrations with CTL
+
+WSO2 API Controller, **apictl** allows you to monitor the synapse artifacts (deployed in a specified Micro Integrator server) and perform various management and administration tasks from the command line. **apictl** communicates with the management API of WSO2 Micro Integrator to function.
+
+!!! info
+    **Before you begin** 
+
+    -   Make sure WSO2 API CTL Tool is initialized and running, if not follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool).
+
+    -  Make sure to add the micro integrator to an environment before you start working with the following CTL commands.      
+    For more information, visit [Add an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#add-an-environment).
+
```

Comment:
Let's also add that you need to ensure that the micro integrator has been started to the **Before you begin** section

Reply:
Fixed with adb71b9